src/v1/services/ping/gets.js

```diff
@@ -0,0 +1,23 @@
+/************
+ * REQUIRES *
+ ************/
+
+// Response
+import DotnetResponses from "dotnet-responses";
+const { Success } = DotnetResponses;
+
+
+
+
+
+/********
+ * GETS *
+ ********/
+
+export function getPing(req, res)
```

Comment:
All of these ping service functions are super simple. They're being separated out purely for the sake of providing an example of how a service layer might look for anyone implementing this template.